Source/Scene/ModelExperimental/ModelMatrixUpdateStage.js

```diff
@@ -67,13 +70,11 @@ function updateRuntimeNode(runtimeNode, sceneGraph, transform) {
       const childRuntimeNode =
         sceneGraph._runtimeNodes[runtimeNode.children[i]];
 
-      Matrix4.multiplyTransformation(
-        runtimeNode.transform,
-        childRuntimeNode.transform,
-        transformScratch
-      );
+      // Update transformToRoot to accommodate changes in the transforms of this node and its ancestors
+      Matrix4.clone(sceneGraphTransform, childRuntimeNode._transformToRoot);
```

Comment:
Again we should update these functions that use `result` to also assign the result to be consistent with the style guide.